README.md

```diff
@@ -63,11 +63,12 @@ You can use the `typescript` preset to get reasonable defaults (it includes the
 <!-- THIS SECTION IS AUTOMATICALLY GENERATED, PLEASE RUN: `npm run build-doc` -->
 
 <!--DOC_START-->
-
 - [`dangerously-set-inner-html`](#dangerously-set-inner-html)
```

Comment:
Sorry, you'll need to rebase. I updated the README file after I saw these unrelated changes here.

Reply:
Rebased. Didn't change.